security/cert_manager_operator/cert-manager-securing-routes.adoc

```diff
@@ -0,0 +1,18 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cert-manager-securing-routes"]
+= Securing OpenShift routes with the {cert-manager-operator}
+include::_attributes/common-attributes.adoc[]
+:context: cert-manager-securing-routes
+
+toc::[]
+
+In the {product-title} 4.16 release, the route API is extended to provide a configurable option to reference TLS certificates via secrets. With the "Configure routes with externally managed certificate" feature enabled, you can minimize errors from manual intervention, streamline the certificate management process, and enable the {product-title} router to promptly serve the referenced certificate.
```

Comment:
/cc @chiragkyal @swghosh 
In case you have any wording suggestions here.^^

Do we need to mention:
1. "In the 4.16 release",
2. it is a Technology Preview feature. (by adding a note like [this](https://docs.openshift.com/container-platform/4.16/networking/routes/secured-routes.html#nw-ingress-route-secret-load-external-cert_secured-routes)?

Reply:
I can add the note for tech preview, and since we are cherry-picking the feature for 4.16+ versions, let me remove the version in the sentence.